sdcm/utils/raft/__init__.py

```diff
@@ -9,9 +9,11 @@
 from sdcm.sct_events.database import DatabaseLogEvent
 from sdcm.sct_events.filters import EventsSeverityChangerFilter
 from sdcm.sct_events import Severity
+from sdcm.utils.version_utils import ComparableScyllaVersion
 
 
 LOGGER = logging.getLogger(__name__)
+RAFT_DEFAULT_SCYLLA_VERSION = "5.5"
```

Comment:
Probably it should be `5.4`.
If not, and `5.5` is expected to be here.
Then, need to use some `dev` version of it, because when you will run a `5.5.0-dev*` version it will be smaller than `5.5` on the line 321 you add below.

Reply:
thanks will fix

Reply:
fixed